code/controllers/subsystem/shuttles.dm

```diff
@@ -254,7 +254,7 @@ var/datum/subsystem/shuttle/SSshuttle
 	slip.ordernumber = ordernum
 
 	var/stationName = (errors & MANIFEST_ERROR_NAME) ? new_station_name() : station_name()
-	var/packagesAmt = SSshuttle.shoppinglist.len + ((errors & MANIFEST_ERROR_COUNT) ? 0 : rand(1,2))
```

Comment:
This is sorta confusing though. MANIFEST_ERROR_COUNT is meant to indicate there's an erroneous count.
So it's actually the other checks which are wrong.